src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml

```diff
@@ -142,7 +142,7 @@
 				Value="Compact" />
 		<Setter Property="ExitDisplayModeOnAccessKeyInvoked"
 				Value="False" />
-		<Setter Property="CornerRadius"
+		<not_win:Setter Property="CornerRadius"
```

Comment:
trying to understand why we need "not_win" here, can you give me a little more details please ?